isolate_example/lib/page_three.dart

```diff
@@ -131,11 +134,18 @@ class DataTransferIsolateController extends ChangeNotifier {
       for (int j = 0; j < 1000000; j++) {
         randNums.add(rng.nextInt(100));
       }
-      await sendNumbers(randNums);
+
+      if (ttd) {
+        final transferable =
```

Comment:
Nit: This could be one line

Reply:
I agree but the Travis Script only passes if I use auto formatting which makes this two lines.

Reply:
Ah I see! I heard about this offline. For the future, if I recommend something that Travis disagrees with, you can ignore me. :) This repository and the flutter/flutter repository use different style guides.